packages/transport/src/modules/sync/externalEventProcessors/ExternalEventProcessorRegistry.ts

```diff
@@ -2,16 +2,15 @@ import { TransportError } from "../../../core";
 import { ExternalEventProcessorConstructor } from "./ExternalEventProcessor";
 import { MessageDeliveredExternalEventProcessor } from "./MessageDeliveredExternalEventProcessor";
 import { MessageReceivedExternalEventProcessor } from "./MessageReceivedExternalEventProcessor";
-import { RelationshipChangeCompletedExternalEventProcessor } from "./RelationshipChangeCompletedExternalEventProcessor";
-import { RelationshipChangeCreatedExternalEventProcessor } from "./RelationshipChangeCreatedExternalEventProcessor";
+import { RelationshipCreatedOrChangedExternalEventProcessor } from "./RelationshipCreatedExternalEventProcessor";
 
 export class ExternalEventProcessorRegistry {
     private readonly processors = new Map<string, ExternalEventProcessorConstructor>();
     public constructor() {
         this.registerProcessor("MessageReceived", MessageReceivedExternalEventProcessor);
         this.registerProcessor("MessageDelivered", MessageDeliveredExternalEventProcessor);
-        this.registerProcessor("RelationshipChangeCreated", RelationshipChangeCreatedExternalEventProcessor);
-        this.registerProcessor("RelationshipChangeCompleted", RelationshipChangeCompletedExternalEventProcessor);
+        this.registerProcessor("RelationshipCreated", RelationshipCreatedOrChangedExternalEventProcessor);
+        this.registerProcessor("RelationshipStatusChanged", RelationshipCreatedOrChangedExternalEventProcessor);
```

Comment:
same processor for different events? could explode
<img src="https://media2.giphy.com/media/KbeMMTRmTCF11CPLq1/giphy.gif"/>

Reply:
the edge case that a status change event arrives before its creation event means that there is some duplication going on - so I went all out on it :)

Reply:
@tnotheis you should look into that

Reply:
shouldn't happen imo

Reply:
discussed with @tnotheis. RelationshipCreated event will be removed. We leave it here for now.